docs/Grafana Dashboard.json

```diff
@@ -145,8 +116,9 @@
                 }
               ],
               "groupByTags": [],
+              "hide": false,
```

Comment:
I can't help but notice that some changes are simply moving lines around, like `hide` and `interval`. Are these changes automated or manual?